test/blackbox-tests/test-cases-with-libs/dune-action-dependency-rebuilt-but-not-changed/bin/dune

```diff
@@ -0,0 +1,3 @@
+(executables
```

Comment:
By the way, (sorry, I should have told you earlier), we've started to use the idiom where we put the dune file into the `.t` file using a bash "here document". For an example see e.g. `test/blackbox-tests/test-cases/cinaps/run.t`. That lets you write entire test in one file, which can be nice.

I'm not saying you should rewrite all tests in this style, just a thing you could try if you want.

Reply:
(sorry, I put this review comment in the wrong dune file, but you get the idea)